.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.csv binary
```

Comment:
Should CSV files really be binary? In my opinion they're human readable/writable and it would be nice to be able to see what changed in a commit.